test/disk-stats-spike/run.sh

```
@@ -0,0 +1,98 @@
+#       docker run --rm --network kind -v "$PWD:/work" -e KUBECONFIG=/work/kc-internal \
+#         alpine/k8s:1.31.2 sh -c '
+#           git clone --depth 1 https://github.com/kubernetes-csi/csi-driver-host-path /csi
+#           cd /csi/deploy/kubernetes-1.30 && bash ./deploy.sh'
+#       kubectl apply -f - <<<"$(printf 'apiVersion: storage.k8s.io/v1\nkind: StorageClass\nmetadata: {name: csi-hostpath-sc}\nprovisioner: hostpath.csi.k8s.io\nvolumeBindingMode: Immediate\n')"
+#     (delete kc-internal afterwards — it embeds cluster certs.)
+#
+# Requires: kind + kubectl + docker, and csi-hostpath-sc pre-installed (above).
+# Run:  bash test/disk-stats-spike/run.sh
+set -euo pipefail
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CLUSTER="${KIND_CLUSTER:-disk-spike}"
```

Shellcheck (0.11.0)

[warning] CLUSTER appears unused. Verify use (or export if used externally).

(SC2034)